src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+/**
+ * Prints necessary output to user.
+ */
+public class Ui {
+    public static final String LINE = "____________________________________________________________\n";
+
+    /**
+     * Introduces bot with welcome message.
+     */
+    public static void showWelcome() {
+        String logo = " ____        _\n"
+                + "|  _ \\ _   _| | _____\n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        String initialPrinting = logo + "Hello! I'm Duke\n" + "What can I do for you?\n"
+                + "____________________________________________________________\n";
+        System.out.println(initialPrinting);
+    }
+
+    /**
+     * Signs off with Bye message.
+     */
+    public static void showByeMessage() {
+        System.out.println("Byeeeee see you later!\n" + LINE);
+    }
+
+    /**
+     * Prints message to user.
+     *
+     * @param message the message to be printed.
+     */
+    public static void printMessageToUser(String message) {
```

Comment:
Perhaps this method can be simply named as printMessage() since it is quite obvious that the message is for the users? This will also be more consistent with the other print..() methods below.